tools/util_linux.go

```diff
@@ -20,7 +10,7 @@ import (
 )
 
 const (
-	ioctlFiClone = C.FICLONE
+	ioctlFiClone = 0x40049409
```

Comment:
It might be valuable for future maintainers if this came with a comment akin to the one removed above from the `import "C"` section, just explaining where it comes from ... I realize the name of the variable provides the context on its own, but I figure the link to the Linux source would ease comprehension too.

Reply:
Sure, I can do that.

Reply:
Done.